opts/opts.go

```diff
@@ -130,6 +172,15 @@ func (opts *ListOpts) Len() int {
 type ValidatorFctType func(val string) (string, error)
 type ValidatorFctListType func(val string) ([]string, error)
 
+func ValidateLogOpts(val string) (string, error) {
+	allowedKeys := map[string]string{}
+	vals := strings.Split(val, "=")
+	if allowedKeys[vals[0]] != "" {
```

Comment:
Sorry I don't get this line: `allowedKeys` is always a newly constructed map, so what's the point of this test?